### ISSUE TITLE
NADfinder requires too much memory

### ISSUE THREAD

**Issue by evangeloo** [jianhong/NADfinder#1]
Hello,

I have been trying to use NADfinder to call peaks for my samples. In total I have 4 bam files, two nucleolar and two whole-genome. Each bam file is ~9GB after dedup and sorting.

I am working with 100GB of RAM. When I perform tileCount the system runs out of memory. The parameters I used are below:

se <- tileCount(reads=file.path(path, f), 
                genome=Hsapiens, 
                windowSize=ws, 
                step=step)  

Is there a variable I can change to more efficiently read in the bam files?

Thanks,
Evan

**Comment by jianhong** [comment]
Yes, this is a know issue. Could please try https://github.com/jianhong/HMMtBroadPeak to call broad peaks?

**Comment by evangeloo** [comment]
Thank you, I will give this a try and reply back with how it goes!

**Comment by evangeloo** [comment]
That worked, thank you so much.
